requirements.txt

```diff
@@ -1,4 +1,5 @@
-conda-forge-metadata==0.8.1
-git+https://github.com/channel-mirrors/conda-oci-mirror.git@v0.1.0
-streamlit==1.37.1
+conda-forge-metadata==0.10.0
+git+https://github.com/channel-mirrors/conda-oci-mirror.git@25ea3e436f0b0bc5a9c646121efafc9c68e116cd # v0.1.0
+streamlit==1.38.0
 streamlit-searchbox==0.1.14
+py-rattler==0.7.0
```

Comment:
Don't you need to add pydantic etc here as well?

Reply:
Thanks for catching this.
02ed25e4a0406e3e1c5eeb847b8c7b98271a52ce